spacer/tasks.py

```diff
@@ -49,7 +50,7 @@ def train_classifier(msg: TrainClassifierMsg) -> TrainClassifierReturnMsg:
     # Do the actual training
     clf, val_results, return_message = trainer(
         ImageLabels.load(msg.traindata_loc),
-        ImageLabels.load(msg.traindata_loc),
+        ImageLabels.load(msg.valdata_loc),
```

Comment:
@qiminchen : I'm integrating spacer in CoralNet and stumbled across this rather nasty bug. Wanted to let you know in case you are using this for your work.

Reply:
thanks for letting me this, all good